CS342MemberViews/Sources/CaitlinKunchur/ReadsRow.swift

```diff
@@ -0,0 +1,93 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import InstructorViews
+import SwiftUI
+
+
+public struct ReadsRow: View {
+    public var body: some View {
+        ZStack {
+            Color(hue: 0, saturation: 0.0, brightness: 1)
+            VStack(alignment: .leading) {
+                Text("Recent Reads").font(.custom(
+                    "GillSans",
+                    fixedSize: 25))
+                    .padding(.leading, 15)
+                    .padding(.top, 5)
+                    .foregroundStyle(.blue.gradient)
+                
+                ScrollView(.horizontal, showsIndicators: false) {
+                    HStack {
+                        VStack {
```

Comment:
Similar to the `HobbiesRow.swift`, I would suggest moving the views into smaller chunks or using a `ForEach` loop to create your views.

Reply:
Hi Paul! Thanks for the feedback! Vishnu and I were able to implement basically all the changes you suggested above! I fixed as many of the SwiftLint errors as I could and just have these two showing up on my end:
<img width="542" alt="image" src="https://user-images.githubusercontent.com/67437371/215292314-d7222922-2d19-4f66-b6d4-0ed4b641761d.png">

However, after committing and pushing, I still see more than just these two swift lint errors on Github's end. I am also not sure how to resolve the "exclude" error because I do have a file called "CaitlinKunchur-bio.md.license"